frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js

```diff
@@ -4,23 +4,11 @@ import { useTranslation } from "react-i18next";
 import { useRouteMatch } from "react-router-dom";
 import CitizenApp from "./pages/citizen";
```

Comment:
Your package.json says the source file is Module.js
https://github.com/Osamai8/DIGIT-OSS/blob/91076f3e80166605324f25c02ee4b829ccbcf4e8/frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/package.json#L8

But the file name was modules.js which made it not to get compiled